assets/src/components/PreviewBanner/PreviewBanner.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import AlertBanner from '../AlertBanner'
+
+const PreviewBanner = props => {
+  const { isDisabled } = props
+
+  if (isDisabled) {
+    return (<AlertBanner>Preview Mode: This view is currently disabled for students.</AlertBanner>)
+  } else {
+    return null
+  }
+}
```

Comment:
I don't really see a problem with this, but I'm not sure it's really worth another component now that I look at it.

Instead of this, you could just have this line in each view:

```
{disabled ? <AlertBanner>Preview Mode: This view is currently disabled for students.</AlertBanner> : undefined}
```

Reply:
Sure.  Is it better to have that exact thing 4 times or have a component that can be changed in one place?

Reply:
I'm ambivalent and I guess it doesn't matter that much.

Reply:
I removed the component and the folders and the lazy stuff.

Reply:
Yeah, the repetition kinda bugs me, but it also didn't feel like the `PreviewBanner` was doing enough to warrant a whole new component. Hm... Anyways, what you currently have is fine for now -- maybe in the future we can determine all the things these views have a common and extract those somehow.